doc/source/development/contributing_environment.rst

```diff
@@ -246,3 +187,25 @@ See https://code.visualstudio.com/docs/remote/containers for details.
 Enable Docker support and use the Services tool window to build and manage images as well as
 run and interact with containers.
 See https://www.jetbrains.com/help/pycharm/docker.html for details.
+
+Step 3: build and install pandas
+--------------------------------
+
+You can now run::
+
+   # Build and install pandas
+   python setup.py build_ext -j 4
+   python -m pip install -e . --no-build-isolation --no-use-pep517
+
+At this point you should be able to import pandas from your locally built version::
+
+   $ python
+   >>> import pandas
+   >>> print(pandas.__version__)  # note: the exact output may differ
+   2.0.0.dev0+880.g2b9e661fbb.dirty
+
+This will create the new environment, and not touch any of your existing environments,
+nor any existing Python installation.
+
+Note that you will need to repeat this step each time the C extensions change, for example
+if you modified them or if you did a fetch and merge from ``upstream/main``.
```

Comment:
```suggestion
if you modified any file in `pandas/_libs` or if you did a fetch and merge from ``upstream/main``.
```

Reply:
there's also some in `pandas/io` though

Reply:
Oh I didn't know there's some sas cython code there (I think that should be moved to `pandas/_libs` at some point).

Mainly wanted to clarify here what files constitute `C extensions`  (pyx files)

Reply:
OK I've gone with your suggestion (it does say "for example", so it's fine if it's not exhaustive, especially if the sas files should be moved to pandas/_libs too)